docs/clone-sst.md

```diff
@@ -0,0 +1,163 @@
+# State Snapshot Transfer (SST) Method for Percona XtraDB Cluster (PXC) Using Clone
+
+--8<--- "tech-preview.md:4:4"
+
+## SST Method: Clone
+
+The clone plugin is a modern and efficient SST method that leverages MySQL's native cloning capabilities to transfer data from a donor node to a joiner node. It is faster and more resource-efficient compared to traditional methods like xtrabackup or rsync.
+
+## Limitations
+
+Clone limitations are described in [Clone plugin limitations](clone-plugin-limitations.html)
+
+## Key features
+
+| Feature                 | Description                                                 |
+|-------------------------|-------------------------------------------------------------|
+| Efficient data transfer | The clone plugin transfers data at the file level, reducing overhead. |
+| Consistency            | Ensures data consistency between the donor and joiner nodes. |
+| Minimal Downtime       | Reduces the time required for node synchronization.         |
+| Native Integration     | Fully integrated into MySQL, eliminating the need for external tools. |
+
+
+## Prerequisites
+
+The requirements for enabling SST transfers with the Clone plugin are as follows:
+
+* Percona XtraDB Cluster (PXC) version 8.0.41 or later
+
+* Sufficient disk space and network bandwidth for data transfer
+
+* Properly configured PXC cluster with at least one donor node
+
+* NetCat package installed
+
+* Port 4444 is used by the State Snapshot Transfer (SST) process in Percona XtraDB Cluster for data transfer between nodes
```

Comment:
this "Port 4444 is used by the State Snapshot Transfer (SST) process in Percona XtraDB Cluster for data transfer between nodes"
Should not be more like this?
"By default port 4444 is used by the State Snapshot Transfer (SST) process as when using Percona Xtrabackup SST for data transfer between nodes"
Or something like that